schema/schema.go

```diff
@@ -348,36 +348,37 @@ func (schema *Schema) StateVersioning() bool {
 	return stateful
 }
 
-func (schema *Schema) SyncKeyTemplate() string {
+
+func (schema *Schema) SyncKeyTemplate() (syncKeyTemplate string, ok bool) {
 	syncKeyTemplateRaw, ok := schema.Metadata["sync_key_template"]
 	if !ok {
-		return ""
-	}
-	syncKeyTemplate, ok := syncKeyTemplateRaw.(string)
-	if !ok {
-		return ""
+		return
```

Comment:
[Same](https://github.com/cloudwan/gohan/pull/227#discussion_r64899412) for this one. It might be necessary to use `=` instead of `:=` for this to return the intended variable's value.